Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -10922,6 +10926,89 @@ a_Player:OpenWindow(Window);
 			},
 			Inherits = "cEntity",
 		},
+		cExpOrb =
+		{
+			Desc = [[
+				This class represents a experience orb. This entity can be spawned by using cWorld:SpawnExperienceOrb.
+				It's also spawned when a monster is killed or a player is mining coal.
+			]],
+			Functions =
+			{
+				GetAge =
```

Comment:
I guess it would. Perhaps later on it's possible to put it in a separate class which `cPickup` and `cExpOrb` inherit from.